TriangleTestNG/test/java/NegativeDataTriangle.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version= "1.0"?>
+<data>
+    <negative side_a="0" side_b="0" side_c="0"/>
+    <negative side_a="0" side_b="1" side_c="1"/>
+    <negative side_a="1" side_b="0" side_c="1"/>
+    <negative side_a="1" side_b="1" side_c="0"/>
+    <negative side_a="100" side_b="200" side_c="1"/>
+    <negative side_a="1" side_b="100" side_c="200"/>
+    <negative side_a="100" side_b="1" side_c="200"/>
+    <negative side_a="-4" side_b="5" side_c="6"/>
+    <negative side_a="4" side_b="-5" side_c="6"/>
+    <negative side_a="4" side_b="5" side_c="-6"/>
+    <negative/>
+    <negative side_a="3"/>
+    <negative side_b="3"/>
+    <negative side_c="3"/>
+    <negative side_a="3" side_b="2"/>
+    <negative side_b="3" side_c="2"/>
+    <negative side_a="3" side_c="2"/>
+    <negative side_a="-Infinity" side_b="4" side_c="3"/>
+    <negative side_a="Infinity" side_b="4" side_c="3"/>
+    <negative side_a="NaN" side_b="4" side_c="3"/>
+    <negative side_a="4" side_b="-Infinity" side_c="3"/>
+    <negative side_a="4" side_b="Infinity" side_c="3"/>
+    <negative side_a="4" side_b="NaN" side_c="3"/>
+    <negative side_a="4" side_b="3" side_c="-Infinity"/>
+    <negative side_a="4" side_b="3" side_c="Infinity"/>
+    <negative side_a="4" side_b="3" side_c="NaN"/>
+    <negative side_a="4.9E-324" side_b="9" side_c="10"/>
+    <negative side_a="20" side_b="10" side_c="4.9E-324"/>
+    <negative side_a="20" side_b="4.9E-324" side_c="10"/>
+    <negative side_a="1.79769313486231E+308" side_b="9" side_c="10"/>
```

Comment:
What is 1.79769313486231E+308? Is it Double.MAX_VALUE? It's not accepted to specified it in such way